assets/olive-optimizer/general/environments/cpu/context/Dockerfile

```diff
@@ -4,7 +4,7 @@ ENV DEBIAN_FRONTEND=noninteractive
 ENV ACCEPT_EULA=Y
 
 RUN apt-get -y update && apt-get -y upgrade && apt-get install -y libsm6 libxext6

```

Comment:
I don't think you need line 6 and 7. Base image update takes care of those, you can just release new environment version